$NetBSD$

1.
From FreeBSD ports 2.5.4

2.
To fix 
/usr/bin/ld: cannot find -lwsjt_fort_omp: No such file or directory
(info from)
https://www.repo.radio/w4kek/WSJT-X/commit/2e3cec2201c33e16c2057958d0eb65ac0f2144a9?style=split&whitespace=show-all&show-outdated=

--- CMakeLists.txt.orig	2021-12-28 18:31:58.000000000 +0900
+++ CMakeLists.txt	2025-10-12 14:16:36.191438094 +0900
@@ -868,7 +868,7 @@
 # OpenMP
 #
 find_package (OpenMP)
-
+set (OpenMP_C_FLAGS "-fopenmp")
 #
 # fftw3 single precision library
 #
@@ -885,7 +885,7 @@
 check_type_size (CACHE_ALL HAMLIB_OLD_CACHING)
 check_symbol_exists (rig_set_cache_timeout_ms "hamlib/rig.h" HAVE_HAMLIB_CACHING)
 
-find_package (Usb REQUIRED)
+#find_package (Usb REQUIRED)
 
 #
 # Qt5 setup
@@ -915,7 +915,7 @@
 #
 set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra")
 
-set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Werror -Wall -Wextra -fexceptions -frtti")
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -fexceptions -frtti")
 
 if (NOT APPLE)
   set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-pragmas")
@@ -1161,7 +1161,7 @@
 target_link_libraries (encode77 wsjt_fort wsjt_cxx)
 
 add_executable (wsprsim ${wsprsim_CSRCS})
-target_link_libraries (wsprsim ${LIBM_LIBRARIES})
+target_link_libraries (wsprsim "-lm")
 
 add_executable (jt4code lib/jt4code.f90)
 target_link_libraries (jt4code wsjt_fort wsjt_cxx)
@@ -1442,7 +1442,7 @@
   )
 
 target_include_directories (wsjtx PRIVATE ${FFTW3_INCLUDE_DIRS})
-if (APPLE)
+if ((NOT ${OPENMP_FOUND}) OR APPLE)
   target_link_libraries (wsjtx wsjt_fort)
 else ()
   target_link_libraries (wsjtx wsjt_fort_omp)
